RockPaperScissors/MookJjiPpaGame.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  MookJjiPpa.swift
+//  RockPaperScissors
+//
+//  Created by kiwi,bard on 2022/04/20.
+//
+import Foundation
+
+
+struct MookJjiPpaGame {
+    enum Numbers {
+        static let range = 1...3
+    }
+    
+    let rpsGame = RockScissorsPaperGame()
+    let printSorts = PrintSorts()
+    
+    func startGame() {
+        printSorts.printRockScissorsPaperMenu()
+        let userChoiceNumber = inputNumber()
+        
+        switch userChoiceNumber {
+        case 0:
+            printSorts.printEndGame()
+        case 1,2,3:
+            guard let userInput = rpsGame.makeRockScissorsPaperUserSign(userInput: userChoiceNumber)
+            else { return }
+            guard let computerInput = rpsGame.makeRockScissorsPaperComputerSign()
+            else { return }
```

Comment:
요건 컨벤션이긴 한데, 보통 한줄에 쓰거나, 밑에 처럼 쓰는 편인것 같아요!!

```suggestion
            guard let userInput = rpsGame.makeRockScissorsPaperUserSign(userInput: userChoiceNumber) else {
                return
            }
            guard let computerInput = rpsGame.makeRockScissorsPaperComputerSign() else {
                return
            }
```

Reply:
코드가 길어서 else 구문을 밑으로 내렸었는데, 다시 한줄로 바꿔주었습니다🙃

Reply:
개행에도 컨벤션이 존재한답니다 ㅎㅎ
두분이서 합의하고 사용하면 정답은 없습니다

예를들어 두분의 코드를 개행하는데도 여러 방법이 있을거 같아요

이런 매서드가 있다고 할때
```swift
guard let pr = sendPullRequest(projectTitle: "묵찌빠게임", team: ["바드", "키위"], reviewer: "dudu", step: 1) else { return }
```
저는 보통 이렇게 개행하는 편입니다
```swift
guard let pr = sendPullRequest(
    projectTitle: "묵찌빠게임", 
    team: ["바드", "키위"], 
    reviewer: "dudu", 
    step: 1
) else {
    return nil
}
```

지금은 사실 인스턴스이름과 매서드이름을 더하니 너무길어져서..
뭐.. 이렇게도 괜찮은것 같구요? 두분의 컨벤션이 확실하면 좋을것 같네요!!
```swift
guard let userInput = rockScissorsPaperGame
.makeRockScissorsPaperUserSign(userInput: userChoiceNumber) else { return }
```

너무 긴 매서드는 적절히 줄바꿈해주는것도 가독성을 높이는 방법이겠죠?

Reply:
넵 알겠습니다~